tests/acceptance/features/webUIFiles/createFolderEdgeCases.feature

```diff
@@ -18,6 +18,7 @@ Feature: create folder
       | 'सिमप्ले फोल्देर $%#?&@' |
       | '"somequotes1"'          |
       | "'somequotes2'"          |
+      | "\"quote\"d-folders'"    |
```

Comment:
why this change?

Reply:
Just wanted to make sure the `concat` part is doing it's job. I know it's bad to test for the test code itself, but it was a change in a critical part of the test code and a valid requirement.

Reply:
Please make then a separate commit for the `concat` part

Reply:
Done. Created separate PR for this feature file addition.